addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -324,7 +324,7 @@ export class SearchBar extends Component {
      * @param {number} index
      */
     onItemMousemove(focusedIndex) {
-        this.computeState({ focusedIndex });
+        this.state.focusedIndex = focusedIndex;
```

Comment:
not sure about this

Reply:
might cause problem programatically: close a many2one then over last item --> focusedIndex becomes bad